_shared/project/pyproject.toml

```diff
@@ -164,7 +169,7 @@ output-format = "colorized"
 score = "no"
 
 [tool.hdev]
-project_name = "{{ cookiecutter.name }}"
+project_name = "{{ cookiecutter.slug }}"
```

Comment:
Small fix: LMS's hdev project name should be `lms` (`cookiecutter.slug`) not `LMS` (`cookiecutter.name`)